src/Misc/atoms/ActionsBar.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from '../../Form/atoms/Button';
+import ButtonWithIcon, { IButtonWithIcon } from '../../Form/atoms/ButtonWithIcon';
+
+const Container = styled.div``;
+const Title = styled.div`
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+  color: hsl(207, 5%, 57%);
```

Comment:
Minor: Hard coded color?

Reply:
Well there was no value that matched the current css styles but this is a common value, perhaps @atomworks  can recommend the way to handle it. 

![Screen Shot 2022-07-21 at 17 28 17](https://user-images.githubusercontent.com/10409078/180167757-2a899d94-6bc1-4430-9b81-2a35b8d64828.png)

In any case I did verify it looks good in light and dark mode

<img width="656" alt="Screen Shot 2022-07-21 at 17 31 05" src="https://user-images.githubusercontent.com/10409078/180168719-fe603e6f-5225-4035-a56c-98fd44db88e4.png">

<img width="612" alt="Screen Shot 2022-07-21 at 17 31 52" src="https://user-images.githubusercontent.com/10409078/180168753-0730df7b-8659-448f-9f8c-3a454d91e9a4.png">